org/org.yaml

```diff
@@ -828,3 +828,53 @@ orgs:
         privacy: closed
         repos:
           resolution: maintain
+  tektoncd-catalog:
+    admins:
```

Comment:
Should this match tektoncd.admins (+ you for additional bootstrapping)?

Reply:
Follow up on the governing-board member meeting: The `tektoncd-catalog` admins should be the same as `tektoncd` admins. 

Will patch the list after discussing the repo-level admins in the catalog WG later this week.

Reply:
@tektoncd/governing-board. I'm also adding @vinamra28 as the org admin to help bootstrap the the org as we discussed in the Catalog WG today. We could remove ourselves out of the list afterward if needed.